### PR TITLE
OKTA-392945 .NET IDX > Model error messages and expose the i18n key.

### PR DESCRIPTION
**Pull request by andriizhegurov-okta**

### CODE REVIEW

**Review by bryanapellanes-okta — APPROVED** [review]
👍

File: src/Okta.Idx.Sdk/IMessage.cs [hunk 1]
```diff
@@ -16,6 +13,9 @@ namespace Okta.Idx.Sdk
     public interface IMessage : IResource
     {
         string Text { get; }
+
         string Class { get; }
```
Comment (laura-rodriguez) [inline src/Okta.Idx.Sdk/IMessage.cs:17]: I don't think we should expose the class since that shouldn't be relevant for devs.

**Review by laura-rodriguez — COMMENTED** [review]
Andrii, can you please update the readme with info about i18n usage?

File: src/Okta.Idx.Sdk/IMessage.cs (hunk 1 above)
Comment (andriizhegurov-okta) [reply src/Okta.Idx.Sdk/IMessage.cs:17]: the Class can be `INFO` or `ERROR`, maybe something else, it's not completely useless. Do you think it's better to hide this info?

Comment (laura-rodriguez) [reply src/Okta.Idx.Sdk/IMessage.cs:17]: To me, this seems to be exclusively for the SIW. I think we shouldn't expose anything related to the UI.

File: README.md [hunk 2]
```diff
@@ -185,6 +185,34 @@ The SDK throws an `OktaException` everytime the server responds with an invalid
 
 `TerminalStateException`  is an `OktaException` derived class that indicates that the user cannot continue the current flow, possibly due to an error or required additional actions outside of the authentication flow.
 
+This exception object contains an array of messages that can be shown to the user as they are. Each message object in the array also contains a key property that can be used for internationalization. 
+Here is an example of accessing the the exception data. All the properties can be null, null checks are not shown in the example.
```
Comment (laura-rodriguez) [inline README.md:189]:
```suggestion
Here is an example of accessing the exception data. All the properties can be null. Null checks are not shown in the example.
```

**Review by laura-rodriguez — APPROVED** [review]

**Pull request merged**